<launch>
  <arg name="image" default="/stereo/left/image_rect_color" />

  <arg name="detection_threshold" default="0.5" />
  <arg name="min_boxes" default="3" />
  <arg name="nms_eps" default="0.2" />
  <arg name="visualize" default="false" />

  <arg name="pretrained_weights" default="$(find fcn_detector)/models/3class_detector/detector.caffemodel" />
  <arg name="deployment_prototxt" default="$(find fcn_detector)/models/3class_detector/deploy.prototxt" />
  <arg name="class_list" default="$(find fcn_detector)/models/3class_detector/class_list.txt" />

 <node pkg="fcn_detector" type="object_detector.py"
       name="object_detector" output="screen" >
   <remap from="image" to="$(arg image)" />
   <param name="pretrained_weights" value="$(arg pretrained_weights)" />
   <param name="deployment_prototxt" value="$(arg deployment_prototxt)" />
   <param name="class_list" value="$(arg class_list)" />

   <param name="detection_threshold" value="$(arg detection_threshold)" />
   <param name="min_boxes" value="$(arg min_boxes)" />
   <param name="nms_eps" value="$(arg nms_eps)" />
   <param name="visualize" value="$(arg visualize)" />
 </node>
</launch>
